src/commands/schema/import/handler.js

```diff
@@ -1,74 +1,120 @@
 const fs = require('fs');
 const pick = require('lodash/pick');
-const findIndex = require('lodash/findIndex');
-const find = require('lodash/find');
+const pkg = require('../../../../package.json');
```

Comment:
I think u can pass `pkg` to `context` in the `src/index.js` and than inject it to the `ctx` in the `src/middlewares/context.js`